cardano-cli/src/Cardano/CLI/Parsers.hs

```diff
@@ -26,9 +26,9 @@ opts =
   Opt.info (parseClientCommand <**> Opt.helper)
     ( Opt.fullDesc
       <> Opt.header
-      "cardano-cli - utility to support a variety of key\
-      \ operations (genesis generation, migration,\
-      \ pretty-printing..) for different system generations."
```

Comment:
While you're here, can you remove multiline literals from these lines and use semigroup concatenation of strings instead?

Reply:
See https://github.com/input-output-hk/cardano-node/issues/4585